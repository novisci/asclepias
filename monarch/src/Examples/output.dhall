List { concepts : Text, facts : < WasBefore | WasAfter >, begin : Integer, end : Integer}
