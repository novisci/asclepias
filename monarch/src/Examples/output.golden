"[[[0,4],{\"getConcepts\":[\"home\"],\"getFacts\":\"WasAfter\",\"getSource\":null}],[[1,10],{\"getConcepts\":[\"not home\"],\"getFacts\":\"WasAfter\",\"getSource\":null}],[[0,20],{\"getConcepts\":[\"home\"],\"getFacts\":\"WasAfter\",\"getSource\":null}]]"